lxd/network/acl/acl_ovn.go

```diff
@@ -1131,3 +1148,53 @@ func ovnParseLogEntry(input string, prefix string) string {
 
 	return string(out)
 }
+
+// ovnParseLogEntriesFromSyslog reads the OVN log entries from the systemd journal and returns them as a list of string entries.
+func ovnParseLogEntriesFromSyslog(l logger.Logger, systemdUnitName string, prefix string) ([]string, error) {
+	var logEntries []string
+	res, err := shared.RunCommand(
+		"/usr/bin/journalctl",
+		"--unit", systemdUnitName,
+		"--directory", "/var/lib/snapd/hostfs/var/log/journal",
+		"--no-pager",
+		"--grep", "acl_log",
```

Comment:
The idea was to remain consistent with the logic inside the `ovnParseLogEntry` that first filter the bulk of the entries with `acl_log` and then with `prefix` (so that we're really sure we're getting acl logs and not a random log that might contain `prefix` in its content... I don't know, maybe this case never happens..)

Reply:
if we can pre-filter it earlier it would make sense to imho

Reply:
ok fixed